MobileWallet/TariLib/Wrappers/Wallet.swift

```diff
@@ -754,14 +741,36 @@ final class Wallet {
         }
     }
 
+    func getBalance() throws -> WalletBalance {
```

Comment:
Avoid usage of `get` and `set` prefixes. Please change this function to `balance()` or replace it with variable (variables can throw errors since Swift 5.5).